global.h

```diff
@@ -94,6 +94,39 @@ enum langid_t {
 
 #define LXRC_WAIT util_wait(__FILE__, __LINE__, __FUNCTION__);
 
+// LOG_* are bitmasks
+#define LOG_LEVEL_SHOW	(1 << 0)
+#define LOG_LEVEL_INFO	(1 << 1)
+#define LOG_LEVEL_DEBUG	(1 << 2)
+// add time stamps to log entries
+#define LOG_TIMESTAMP	(1 << 3)
+// add calling function name to log entries
+#define LOG_CALLER	(1 << 4)
+
+// log to the default console
+#define log_show(...) util_log(LOG_LEVEL_SHOW, __VA_ARGS__)
+// log to the logging console
+#define log_info(...) util_log(LOG_LEVEL_INFO, __VA_ARGS__)
+// log to the log file
+#define log_debug(...) util_log(LOG_LEVEL_DEBUG, __VA_ARGS__)
+// log to the default console if cond is true
+// else log to logging console
+#define log_show_maybe(cond, ...) util_log((cond) ? LOG_LEVEL_SHOW : LOG_LEVEL_INFO, __VA_ARGS__)
```

Comment:
btw one advantage of suggested name is also that it shows second log level, which is otherwise hidden in implementation or documentation.